hack/update-vendor.sh

```diff
@@ -240,6 +240,14 @@ for repo in $(tsort "${TMP_DIR}/tidy_deps.txt"); do
 $(go mod why ${loopback_deps})"
       exit 1
     fi
+
+    # prune unused pinned replace directives
+    comm -12 \
```

Comment:
`comm` is a new dependency, do we want to print a good error when this is missing?

Reply:
it's posix, so I'm assuming it's available (like sed, awk, xargs, grep, join, tsort, cat, etc)